checkov/common/checks_infra/solvers/attribute_solvers/base_attribute_solver.py

```diff
@@ -30,6 +30,7 @@
 
 class BaseAttributeSolver(BaseSolver):
     operator = ""  # noqa: CCE003  # a static attribute
+    parsed_attributes = {}
```

Comment:
Shouldn't we init it in the init method?

Reply:
i want to keep it in class level in purpose. if we init this class everytime, it will not improve anything

Reply:
ok ill do another test to see if we can do in on instance level